src/module/migration/migrations/815-spin-tale-adventure-specific.ts

```diff
@@ -0,0 +1,24 @@
+import { ItemSourcePF2e } from "@item/data";
+import { MigrationBase } from "../base";
+
+/** Replace links to adventure-specific Spin Tale */
+export class Migration815SpinTaleAdventureSpecific extends MigrationBase {
+    static override version = 0.815;
+
+    override async updateItem(source: ItemSourcePF2e): Promise<void> {
+        if (source.type === "feat") {
+            const oldSpinTale = "Compendium.pf2e.adventure-specific-actions.Spin Tale";
+            const newSpinTale = "Compendium.pf2e.actionspf2e.Spin Tale";
```

Comment:
Do we want to migrate for `5gahZQXf3UVwATSC` as well? If anyone dragged in the action anywhere that will be how it's listed.